pages/software/modules.md

```diff
@@ -440,9 +443,9 @@ $ R CMD INSTALL -l /projects/academic/[YourGroupName]/$USER/software/$CCR_VERSIO
 
 Sometimes R library installations will fail with errors such as `installation of package ‘library_name’ had non-zero exit status` or similar.  This is a result of the software trying to save temporary files where you do not have permission to write files.  As a workaround, create a temporary directory within your home or project directory and point the R library installation to use it.  As an example:  
 ```
-[login:~]$ mkdir ~/tmp  
-[login:~]$ module load gcc openmpi r  
-[login:~]$ R 
+$ mkdir ~/tmp  
+$ module load gcc openmpi r  
+$ R 
 Sys.setenv(TMPDIR="/user/$USER/tmp")  
```

Comment:
Did we want to prepend ">" to the R REPL lines in this case to delineate shell commands vs. R commands? Might be less confusing, in my opinion.